o1vm/src/pickles/column_env.rs

```diff
@@ -0,0 +1,102 @@
+use ark_ff::FftField;
+use ark_poly::{Evaluations, Radix2EvaluationDomain};
+
+use crate::interpreters::mips::witness::SCRATCH_SIZE;
+use kimchi::circuits::{
+    domains::EvaluationDomains,
+    expr::{
+        BerkeleyChallengeTerm, BerkeleyChallenges, ColumnEnvironment as TColumnEnvironment,
+        Constants, Domain,
+    },
+};
+
+use super::proof::WitnessColumns;
+
+/// The collection of polynomials (all in evaluation form) and constants
+/// required to evaluate an expression as a polynomial.
+///
+/// All are evaluations.
+pub struct ColumnEnvironment<'a, F: FftField> {
+    /// The witness column polynomials. Includes relation columns and dynamic
```

Comment:
is it me or is witness polynomials sometimes used only for the wires, and sometimes for all the polynomials we commit to during the IOP ?

Reply:
Do you mean the term "witness polynomial"?

Reply:
yes

Reply:
Yes, we do use "witness polynomials" for everything. We should try to enforce the distinctions to avoid misunderstandings.